vue-components/stories/OptionsMenu.stories.ts

```diff
@@ -79,12 +74,11 @@ export function withItems( args: { boldLabels: boolean, selectedItemIndex: numbe
 
 withItems.args = {
 	boldLabels: true,
+	menuItems: vegetableItems,
```

Comment:
This control feels a bit cluncky, which makes me question how usable it actually is, maybe we should create a couple of controls for only one item from the list? (i.e. label, description, tag)